### DISCUSSION TITLE
Help Needed with SnapMirror Fields/Metrics Collection

### DISCUSSION THREAD

**Discussion by ryaddaoud21** [NetApp/harvest#3371] [General]
Hello everyone,

I am trying to collect SnapMirror fields using the endpoint **api/private/cli/snapmirror** . According to the documentation, I should receive the following fields ( and others...) :

- destination_path => destination_location
- relationship_id => relationship_id
- cg_item_mappings => cg_item_mappings
- destination_volume => destination_volume
- destination_volume_node => destination_node
- destination_vserver => destination_vserver
- healthy => healthy
- last_transfer_type => last_transfer_type
- policy_type => policy_type
- relationship_group_type => group_type
- relationship_type => relationship_type
- schedule => schedule
- source_path => source_location
- source_volume => source_volume
- source_vserver => source_vserver
- status => relationship_status
- unhealthy_reason => unhealthy_reason
- break_failed_count => break_failed_count
- break_successful_count => break_successful_count
- lag_time(duration) => lag_time
- last_transfer_duration(duration) => last_transfer_duration
- last_transfer_end_timestamp(timestamp) => last_transfer_end_timestamp
- last_transfer_size => last_transfer_size
- newest_snapshot_timestamp(timestamp) => newest_snapshot_timestamp
- resync_failed_count => resync_failed_count
- resync_successful_count => resync_successful_count
- total_transfer_bytes => total_transfer_bytes
- total_transfer_time_secs => total_transfer_time_secs
- update_failed_count => update_failed_count
- update_successful_count => update_successful_count

However, the response I am receiving contains only the following fields:

```
"records": [
    {
        "source_path": "#:#####",
        "source_vserver": "#",
        "source_volume": "#",
        "destination_path": "#:#",
        "destination_vserver": "#",
        "destination_volume": "#"
    }
]
```

Does anyone have any insights on what might be going wrong? BTW I did not have any issues with **aggregation** and the **nic_common** endpoint/fields before.

Thanks in advance for your help!

Best,

**Comment by cgrinds** [comment]
hi @ryaddaoud21 what version of Harvest are you using and can you share your poller log files for the poller in question? You have pasted part of the `conf/rest/9.12.0/snapmirror.yaml` template. Does the poller in question list the Rest or Zapi collector?
https://netapp.github.io/harvest/24.11.1/help/log-collection/

**Reply by ryaddaoud21** [reply]
Hello, thank you for your reply. The polar in question is the REST API, and the version of my harvest in the cluster is 9.12.1.

**Reply by cgrinds** [reply]
Please collect the log files for the poller in question and email them to ng-harvest-files@netapp.com so we can take a look.
